misc/media.talon

```diff
@@ -1,6 +1,7 @@
 volume up: key(volup)
 volume down: key(voldown)
+set volume <number>: user.media_set_volume(number)
```

Comment:
This action is not defined anywhere; did you forget to include it in the push?

Reply:
I did, thank you. I also need to modify the implementation to only apply to mac